web_responsive/static/src/css/web_responsive.scss

```
@@ -576,3 +578,83 @@ html .o_web_client .o_main .o_main_content {
 .oe_wait {
   cursor: progress;
 }
+
+// Document Viewer
+.o_web_client.o_chatter_position_sided {
+    .o_document_viewer {
+        z-index: 10 !important;
+        @include media-breakpoint-up(lg) {
+            .o_maximize_btn, .o_minimize_btn, .o_split_pdf_area {
+                display: inline !important;
+            }
+        }
+        @include media-breakpoint-between(md, lg) {
+            .o_split_pdf_area {
+                display: inline !important;
+            }
+        }
+    }
+    .o_responsive_document_viewer {
+        @include media-breakpoint-up(lg) {
+            width: $chatter_zone_width !important;
+            margin-left: auto;
+
+            &:hover .arrow, &:hover .o_viewer_toolbar {
+                display: unset;
+            }
+
+            .arrow, .o_viewer_toolbar {
+                display: none;
+            }
+
+            .o_viewer_img_wrapper {
+                position: relative !important;
+
+                .o_viewer_pdf {
+                    width: 95% !important;
+                }
+            }
+        }
+    }
+}
+.o_web_client:not(.o_chatter_position_sided) {
+    .o_split_pdf_area {
+        display: inline !important;
+    }
+}
+//-- Generic
```

```suggestion

// Generic
```

BTW what does Generic mean? :thinking: